src/components/BackButton/BackButton.styled.jsx

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+
+ const BackBtnContainer = styled.div`
+  display: flex;
+  cursor: pointer;
+
+`
+const BackBtn = styled.img`
+  @media (max-width: 960px) {
+    height: 100%;
+    width: 33.33vw;
```

Comment:
I think the BackButton can get too big for example in a screen device of 800px, what do you think about add a max-width here or in the container to avoid this problem?

Screenshot
![imagen](https://user-images.githubusercontent.com/66505715/140177196-6d15ec23-be0e-4979-8202-bc4ebecb23f2.png)